interface/usergroup/usergroup_admin_add.php

```diff
@@ -37,98 +58,73 @@ function trimAll(sString)
-          if(f[i].type=='text' && f[i].value)
-          {
-             if(f[i].name == 'rumple')
-             {
-                alertMsg += checkLength(f[i].name,f[i].value,35);
-                alertMsg += checkUsername(f[i].name,f[i].value);
-             }
-             else if(f[i].name == 'fname' || f[i].name == 'mname' || f[i].name == 'lname')
-             {
-                alertMsg += checkLength(f[i].name,f[i].value,35);
-                alertMsg += checkUsername(f[i].name,f[i].value);
-             }
-             else if(f[i].name == 'federaltaxid')
-             {
-                alertMsg += checkLength(f[i].name,f[i].value,10);
-                alertMsg += checkFederalEin(f[i].name,f[i].value);
-             }
-             else if(f[i].name == 'state_license_number')
-             {
-                alertMsg += checkLength(f[i].name,f[i].value,10);
-                alertMsg += checkStateLicenseNumber(f[i].name,f[i].value);
-             }
-             else if(f[i].name == 'npi')
-             {
-                alertMsg += checkLength(f[i].name,f[i].value,35);
-                alertMsg += checkTaxNpiDea(f[i].name,f[i].value);
-             }
-             else if(f[i].name == 'federaldrugid')
-             {
-                alertMsg += checkLength(f[i].name,f[i].value,30);
-                alertMsg += checkAlphaNumeric(f[i].name,f[i].value);
-             }
-          }
-       }
-       if(alertMsg)
-       {
-          alert(alertMsg);
-          return false;
-       }
-       <?php } // End erx_enable only include block?>
-
-        document.forms[0].submit();
-        parent.$.fn.fancybox.close(); 
-
-    } else {
-       if (document.forms[0].rumple.value.length<=0)
-       {
-          document.forms[0].rumple.style.backgroundColor="red";
-          alert("<?php xl('Required field missing: Please enter the User Name','e');?>");
-          document.forms[0].rumple.focus();
-          return false;
-       }
-       if (document.forms[0].stiltskin.value.length<=0)
-       {
-          document.forms[0].stiltskin.style.backgroundColor="red";
-          alert("<?php echo xl('Please enter the password'); ?>");
-          document.forms[0].stiltskin.focus();
-          return false;
-       }
-       if(trimAll(document.getElementById('fname').value) == ""){
-          document.getElementById('fname').style.backgroundColor="red";
-          alert("<?php xl('Required field missing: Please enter the First name','e');?>");
-          document.getElementById('fname').focus();
-          return false;
-       }
-       if(trimAll(document.getElementById('lname').value) == ""){
-          document.getElementById('lname').style.backgroundColor="red";
-          alert("<?php xl('Required field missing: Please enter the Last name','e');?>");
-          document.getElementById('lname').focus();
-          return false;
-       }
-    }
+
+    var valid = submitme(1, undefined, 'new_user', collectvalidation);
```

Comment:
disregard above question. I figured out the code below does get executed on a successful submit()